supervisor_docker.conf

```diff
@@ -40,7 +29,7 @@ startretries=2
 stopsignal=QUIT
 
 [program:qworker]
-command=sh -c "while [ ! -f /tmp/backend_ready ]; do echo 'Waiting for backend to be ready...'; sleep 2; done; if [ \"$RUN_QWORKER_DEV_MODE\" = \"true\" ]; then echo 'Running in qworker DEV mode'; watchfiles --filter python 'python manage.py qcluster' /code/backend; else echo 'Running in qworker PROD mode'; python manage.py qcluster; fi"
```

Comment:
The background task process setup all file are being located in the `background_tasks` folder. so felt if the worker just reload/watches that folder would be sufficient rather than both Unicorn and QWorkers doing it. This is just for development, since `watchfiles` package only used for reloading the development